tools/rostopic/src/rostopic/__init__.py

```diff
@@ -187,6 +187,10 @@ def _rostopic_hz(topic, window_size=-1, filter_expr=None):
     :param window_size: number of messages to average over, -1 for infinite, ``int``
     :param filter_expr: Python filter expression that is called with m, the message instance
     """
+
+    if rospy.get_param('use_sim_time', False):
+        sys.stderr.write("WARNING: topic [%s] may be using simulated time\n" % (topic))
```

Comment:
Can you please use the `print()` function for this (similar as in https://github.com/spaghetti-/ros_comm/blob/46857a2483a38648635740a16b63cc869d9e8177/tools/rostopic/src/rostopic/__init__.py#L305).